synapse/config/server.py

```diff
@@ -652,37 +663,37 @@ class LimitRemoteRoomsConfig:
             )
 
         manhole_settings = config.get("manhole_settings") or {}
-        manhole_username = manhole_settings.get("username") or "matrix"
-        manhole_password = manhole_settings.get("password") or "rabbithole"
-        manhole_priv_key_file = manhole_settings.get("ssh_priv_key") or None
-        manhole_pub_key_file = manhole_settings.get("ssh_pub_key") or None
+        validate_config(_MANHOLE_SETTINGS_SCHEMA, manhole_settings, ())
```

Comment:
I think this is missing the value for the path:

```suggestion
        validate_config(_MANHOLE_SETTINGS_SCHEMA, manhole_settings, ('manhole_settings',))
```